README.md

```diff
@@ -11,22 +11,72 @@ For normal SAFE Network browsing, you should download the latest version of the
 Application developers should use the same link, but choose the `-dev` postfixed version for their platform. This version uses a `mock` network to allow local development (without the need to pay PUT costs on a live SAFE Network).
 
 
-## WebApp Development 
+## WebApp Development
 
-There are `dev-` prefixed releases of Peruse available. These come with both live network and mock network libs, bundled.
+There are `-dev` postfixed releases of Peruse available. These come with both live network and mock network libs, bundled.
 
-By default, opening the app will open Peruse for the mock network (when you're running in a `NODE_ENV=dev` environment).
+By default, opening the app will open Peruse for the mock network.
 
 Otherwise, there is the option to pass a `--live` flag to the browser. This will start the browser in a `live` network mode.
 
 eg, on OSX:
 
-`NODE_ENV=dev open Peruse.app --args --live`
+```bash
+open Peruse.app --args --live
+```
+
+### SAFE Network API
+
+Peruse exposes a set of APIs in the DOM which webapps can make use to connect to the SAFE Network, as well as fetch and store data on it.
+
+A webapp have direct access to this set of APIs thru the DOM at `window.safe`.
+
+The SAFE Network client API exposed by Peruse is a simple wrapper on top of the API provided by the `@maidsafe/safe-node-app` package (with a few minor exceptions explained below), therefore the documentation available for the `safe-node-app` API is valid and t's what a developer creating a webapp for Peruse needs to use as a reference.
```

Comment:
> and t's what a developer creating a webapp for Peruse needs to use as a reference.

`and is the main reference for any developer wanting to create a webapp for The SAFE Network.`